src/runtime/runtime.js

```diff
@@ -580,7 +580,13 @@ export class ConfiguredRuntime {
     preconnect.prefetch('$assets$/loader.svg');
     LinkCompleteFlow.configurePending(this);
     PayCompleteFlow.configurePending(this);
-    this.payClient_.preconnect(preconnect);
+    if (isExperimentOn(this.win_, ExperimentFlags.PAY_CLIENT_LAZYLOAD)) {
```

Comment:
These 3 preconnects should always happen regardless of pay. They are independent/unrelated.

Reply:
Good point. I'll just move these preconnects directly into Runtime now.